### DISCUSSION TITLE
how to use tracking for only one class?

### DISCUSSION THREAD

**Discussion by MikhailOznobikhin** [mikel-brostrom/yolo_tracking#697] [Q&A] [answered]
hello, tell me, please, is it possible to somehow use tracking only on class 0, and the rest can simply be recognized using yolo?

**Comment by mikel-brostrom** [comment, marked as answer]
`python track.py --source 0 --yolo-weights yolov8s.pt --classes 17  # COCO yolov8 model. Track dogs, only`

**Reply by MikhailOznobikhin** [reply]
thanks! But why yolo detect more object? Can i fix it without using tracking on them
